contracts/BridgeAdmin.sol

```diff
@@ -1,13 +0,0 @@
-// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
I removed this for now...we dont have a definition of what this requires just yet and I thought it cleaner to directly add the finder to the hub.